NuGet.config

```diff
@@ -4,6 +4,7 @@
     <clear />
     <add key="NuGet" value="https://api.nuget.org/v3/index.json" />
     <add key="Esprima" value="https://www.myget.org/F/esprimadotnet/api/v3/index.json" />
+    <add key="Local" value="libs" />
```

Comment:
left-over?

Reply:
Still needed for NUnit local lib

Reply:
No longer needed, using nuget.org package now